packages/babel-parser/src/parser/error-message.js

```diff
@@ -95,6 +99,8 @@ export const ErrorMessages = Object.freeze({
     "Only string literals are allowed as module attribute values",
   ModuleAttributesWithDuplicateKeys:
     'Duplicate key "%0" is not allowed in module attributes',
+  ModuleExportNameHasLoneSurrogate:
+    "Export name can not include lone surrogate, but '\\u%0' is seen",
```

Comment:
Maybe something like?

```suggestion
    "An export name cannot be a lone surrogate, found '\\u%0'",
```

Reply:
A lone surrogate refers to a character falling into non-paired surrogates. An export name is a string thus conceptually an export name can _include_ some character.